### PR TITLE
UI: Provide configuration option to use browser location by default.

### PR DESCRIPTION
**Pull request by devaskim**
browser_location

### CODE REVIEW

File: ui-ngx/src/app/modules/home/components/widget/lib/maps/leaflet-map.ts [hunk 1]
```diff
@@ -494,7 +495,26 @@ export default abstract class LeafletMap {
...
+        if (this.options.useBrowserCenterPosition) {
+          const options = {
+            enableHighAccuracy: true,
+            timeout: 1000,
+            maximumAge: 0
+          };
+          navigator.geolocation.getCurrentPosition(
+              position => {
+                this.browserLocation = [position.coords.latitude, position.coords.longitude]
+                this.map.panTo(this.browserLocation)
```
Comment (vvlladd28) [inline ui-ngx/src/app/modules/home/components/widget/lib/maps/leaflet-map.ts:507]: You have lost the map bounds calculation

File: ui-ngx/src/app/modules/home/components/widget/lib/maps/leaflet-map.ts [hunk 2]
```diff
@@ -494,7 +495,26 @@ export default abstract class LeafletMap {
...
+              },
+              () => {
+                if (this.options.useDefaultCenterPosition) {
+                  this.map.panTo(this.options.parsedDefaultCenterPosition);
+                  this.bounds = map.getBounds();
+                }
```
Comment (vvlladd28) [inline ui-ngx/src/app/modules/home/components/widget/lib/maps/leaflet-map.ts:513]: If useDefaultCenterPosition is false, map incorrect init bounds

**Comment by ashvayka** [comment]
@vvlladd28 , Let's try to merge this to 3.5

**Comment by devaskim** [comment]
@ashvayka @vvlladd28 Guys, what is the problem to merge this PR for almost half a year?

**Pull request closed without merging**